recipes/h3/build.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+set -ex
+
+export LDFLAGS="$LDFLAGS -lrt"
```

Comment:
Remove this line. It's not needed anymore

Reply:
@isuruf I applied your suggestion. Now OsX is green as well. Windows still fail with: `No files or script found for output h3
`